app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,65 @@
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all
+  end
+
+  def new # Only cares a bout showing the form
+    @task = Task.new
+  end
+
+  def create # has access to user data, from the form
+    #before, using a hand rolled form, the format of the data coming onto the wire
+    # was different so the way we pulled it out was also different
+    #book = Book.new(title: params[:title] ,description: params[:description], due_date: params[:due_date], status: params[:status])
+
+    task = Task.new(title: params[:task][:title] ,description: params[:task][:description], due_date: params[:task][:due_date], complete: params[:task][:complete])
+    task.save
+    redirect_to('/tasks')
```

Comment:
Ideally here we would use `redirect_to tasks_path` instead. It should work the same way as this line does, but is more flexible if we for some reason need to change the URL that maps to the `Tasks#index` action.